schema/sale_artwork.js

```diff
@@ -99,17 +99,13 @@ const SaleArtwork = {
   type: SaleArtworkType,
   description: 'A Sale Artwork',
   args: {
-    sale_id: {
```

Comment:
Nah this is fine; nothing's using it yet and I was going to make a similar change. Much appreciated